### PR TITLE
Python Flask/Ruby Sinatra examples

### PR DESCRIPTION
**Pull request by LeonarthusMectus**
This PR adds server example implementations for both standard & advanced integrations.

* Ruby was implemented using the Sinatra framework for both standard & advanced
* Python was implementation using the Flask framework for both standard & advanced

The HTML front end was used to validate both language backends in both standard & advanced integrations as per the instructions provided in the repository including the PayPal flow + card fields flow (for advanced).

### CODE REVIEW

File: advanced-integration/v2/server/python/server/__init__.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import logging
```
Comment (amsourav) [inline advanced-integration/v2/server/python/server/__init__.py:1]: can we keep the content directly in a `server.py` file
so the new path for this file would be - `advanced-integration/v2/server/python/server.py`

File: standard-integration/server/ruby/Gemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "dotenv", "~> 3.1"
+gem "paypal-server-sdk", "~> 0.5.1"
+gem "puma", "~> 6.4"
+gem "rackup", "~> 2.1"
```
Comment (amsourav) [inline standard-integration/server/ruby/Gemfile:8]: we can remove rackup and puma

Comment (LeonarthusMectus) [reply standard-integration/server/ruby/Gemfile:8]: No, these are required by Sinatra.

File: advanced-integration/v2/server/python/server/__init__.py (hunk 1 above)
Comment (LeonarthusMectus) [reply advanced-integration/v2/server/python/server/__init__.py]: Done.

File: standard-integration/server/ruby/server.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          amount: AmountWithBreakdown.new(
+            currency_code: 'USD',
+            value: '100.00'
+          )
+        )
+      ]
+    ),
+    'prefer' => 'return=representation'
+    })
+  order_response.data.to_json
```
Comment (amsourav) [inline standard-integration/server/ruby/server.rb:57]: @LeonarthusMectus can you return this response as json, similar to Line 35. By default `text/html;charset=utf-8` will be return instead

Comment (LeonarthusMectus) [reply standard-integration/server/ruby/server.rb]: @amsourav Good catch, fixed this up in both standard & advanced integrations.

**Review by amsourav — APPROVED** [review]

**Review by sarat-a — COMMENTED** [review]
@LeonarthusMectus - Can we remove .env usage and stick to Environment Variables like used in other languages? https://github.com/paypal-examples/docs-examples/blob/main/advanced-integration/v2/server/dotnet/README.md

File: advanced-integration/v2/server/python/server.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+from paypalserversdk.controllers.orders_controller import OrdersController
+from paypalserversdk.models.amount_with_breakdown import AmountWithBreakdown
+from paypalserversdk.models.checkout_payment_intent import CheckoutPaymentIntent
+from paypalserversdk.models.order_request import OrderRequest
+from paypalserversdk.models.purchase_unit_request import PurchaseUnitRequest
+from paypalserversdk.api_helper import ApiHelper
+
+app = Flask(__name__)
+
+load_dotenv()
```
Comment (amsourav) [inline advanced-integration/v2/server/python/server.py:19]: I think we can remove the dotenv reference as the README steps for other languages direct the user to explicitly add the environment variables in their shell. 

ref: https://github.com/paypal-examples/docs-examples/blob/c964b7083a62e76886251f34dd822fb02258b3c6/advanced-integration/v2/server/dotnet/README.md#running-the-sample

Comment (LeonarthusMectus) [reply advanced-integration/v2/server/python/server.py]: Removed the .env files and update the README to match.  

I also caught at the last moment that Integration appears to be misspelled as `Integartion` in all the server README files.  I've corrected it here for the new additions which were based on the existing READMEs, but thought I'd point that out.

File: .gitignore [hunk 5]
```diff
@@ -101,6 +101,7 @@ dist
 .tern-port
 
 # dotnet
+<<<<<<< HEAD
```
Comment (amsourav) [inline .gitignore:104]: Can you please remove this

File: advanced-integration/v2/server/python/server.py (hunk 4 above)
Comment (amsourav) [reply advanced-integration/v2/server/python/server.py]: Thanks @LeonarthusMectus, It has been fixed in other languages

**Review by sarat-a — APPROVED** [review]

**Review by gregjopa — APPROVED** [review]

**Pull request merged**